OpenRA.Game/Graphics/SpriteCache.cs

```diff
@@ -73,6 +73,11 @@ static ISpriteFrame[] GetFrames(IReadOnlyFileSystem fileSystem, string filename,
 			}
 		}
 
+		public ISpriteFrame[] LoadFramesUncached(string filename)
+		{
+			return GetFrames(fileSystem, filename, loaders, out _);
```

Comment:
Maybe `GetFrames` should remove the `out TypeDictionary metadata` parameter since neither caller uses it?